### ISSUE TITLE
Importing library causes errors

### ISSUE THREAD

**Issue by klische** [ProtonMail/gopenpgp#30]
When I import `github.com/ProtonMail/gopenpgp/crypto`, I get the error `module golang.org/x/crypto@latest found (v0.0.0-20191119213627-4f8c1d86b1ba), but does not contain package golang.org/x/crypto/rsa`

So I noticed another issue https://github.com/ProtonMail/gopenpgp/issues/8, that mentioned having the replace directive from the go.mod file in our go.mod file, but that results in this error:

```
not enough arguments in call to gomime.DecodeCharset
	have ([]byte, map[string]string)
	want ([]byte, string, map[string]string)
```

Can someone please help me fix this? How are you importing the library in Go (using modules)?

**Comment by antongocode** [comment]
The API of the transitive dependency `github.com/ProtonMail/go-mime`, change since the v1.0.0 release of gopenpgp. Add the entry in this project's go.mod file to your own. It fixes the issue.
`github.com/ProtonMail/go-mime v0.0.0-20190923161245-9b5a4261663a // indirect`

**Comment by Syleron** [comment]
@antongocode With this fix I now get:

`undefined: "github.com/ProtonMail/gopenpgp/crypto".GenerateKey`

In attempt to use the GenerateKey function. Any ideas?

**Comment by antongocode** [comment]
@Syleron The documentation is for the latest code not the v1.0.0 release. The global GenerateKey (and others) function does not exist in the release. For the release you need to instantiate an instance of the GopenPGP struct. Here's an example.

```
package foo

import (
	"testing"
	gopenpgp "github.com/ProtonMail/gopenpgp/crypto"
)

var pgp = gopenpgp.GetGopenPGP()

func TestKeygen(t *testing.T) {
    const (
		localPart  = "name.surname"
		domain     = "example.com"
		passphrase = "LongSecret"
		rsaBits    = 2048
		ecBits     = 256
	)
	_, _ = pgp.GenerateKey(localPart, domain, passphrase, "rsa", rsaBits)
}
```

**Comment by Syleron** [comment]
@antongocode Ah I see, great thank you.

**Comment by wussler** [comment]
The module `go-pm-mime` has been updated with V2

**Comment by klische** [comment]
Sorry for not responding earlier, this has been fixed :) Also tested with v2.0.1 and confirmed working there.

**Comment by mwmahlberg** [comment]
> github.com/ProtonMail/go-mime v0.0.0-20190521135552-09454e3dbe72 // indirect

Actually, as of today, it does not.
